unified_full_viewing_keys.py

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env python3
+import sys; assert sys.version_info[0] >= 3, "Python 3 required."
+
+import math
+from random import Random
+import struct
+
+from bech32m import bech32_encode, bech32_decode, convertbits, Encoding
+
+from tv_output import render_args, render_tv, Some
+from tv_rand import Rand, randbytes
+from zc_utils import write_compact_size, parse_compact_size
+from f4jumble import f4jumble, f4jumble_inv
+import orchard_key_components
+import sapling_key_components
+import sapling_zip32
+from unified_encoding import encode_unified, decode_unified
+from unified_encoding import P2PKH_ITEM, P2SH_ITEM, SAPLING_ITEM, ORCHARD_ITEM
+
+def main():
+    args = render_args()
+
+    rng = Random(0xabad533d)
+    rand = Rand(randbytes(rng))
+    seed = rand.b(32)
+
+    test_vectors = []
+    for i in range(0, 10):
+        has_t_key = rand.bool()
+        if has_t_key:
+            t_key_bytes = b"".join([rand.b(32), bytes([0x02]), rand.b(32)])
```

Comment:
Randomly choose 0x02 or 0x03; this ensures that the range of simulated key encodings covers the actual range.

Reply:
I've updated the test vectors to use real secp256k1 points.